alws/build_planner.py

```diff
@@ -135,16 +140,88 @@ async def init_build_repos(self):
     async def add_linked_builds(self, linked_build):
         self._build.linked_builds.append(linked_build)
 
-    def remove_force_to_build_artifacts_from_template(
-        self,
-        module: ModuleWrapper,
-        build_task_refs: typing.List[build_schema.BuildTaskRef],
-    ):
-        # if some refs with added artifacts in module template
-        # was enabled to build, we should delete them from template
-        for ref in build_task_refs:
+    async def get_multilib_artifacts(
+            self, task: build_schema.BuildTaskModuleRef,
+            has_devel: bool = False
+    ) -> typing.List[dict]:
+        beholder_client = BeholderClient(
+            settings.beholder_host, token=settings.beholder_token)
+        multilib_artifacts = []
+
+        # FIXME: Hardcoded platform for now, should send actual platform
+        #  in the task ref
+        platform = self._platforms[0]
```

Comment:
if we would build packages/modules for few platforms, there always be only first platform in list
for `platform_name` it would be ok, but not for `major_distr_version`